sidekick_core/lib/src/commands/format_command.dart

```diff
@@ -0,0 +1,203 @@
+import 'package:glob/glob.dart';
+import 'package:glob/list_local_fs.dart';
+import 'package:meta/meta.dart';
+import 'package:sidekick_core/sidekick_core.dart';
+import 'package:yaml/yaml.dart';
+
+/// Formats the Code for all Flutter/Dart packages in the repository
+class FormatCommand extends Command {
+  @override
+  String get description => 'Formatting the code for all packages';
+
+  @override
+  String get name => 'format';
+
+  /// packages whose files should not be formatted
+  final List<DartPackage> exclude;
```

Comment:
I don't think excluding packages is useful, because this command primarily operates on files

Reply:
So we completely remove it?